integration_test/tests/connection_tests.rs

```diff
@@ -109,6 +118,7 @@ mod integration {
     // the logger log level is updated to the connection log level.
     // If you are having problems running this test, ensure you are running as an administrator.
     #[test]
+    #[cfg(feature = "evergreen_tests")]
```

Comment:
This will be great. I was thinking about it over the weekend and remembered this an easy way to do this (from the testing overhaul I did on mongosql).

We can change this declaration here to the following
```suggestion
    #[cfg_attr(not(feature = "evergreen_tests"), ignore)]
```

And then remove all of the other added `[#cfg(..)]` directives you added. This then makes the evergreen file a lot easier.